site/en/docs/webstore/about_webstore/index.md

```diff
@@ -21,13 +21,13 @@ build Chrome-specific features into it.
 
 ## How do I start?
 
-1.  Learn more about [Chrome Extensions][extensions]
+1.  Learn more about [Chrome Extension development][getstarted]
```

Comment:
Per our handbook, please reserve numbered items for procedures. Make this into bullets.

Reply:
I'm going to do this on a separate PR 👍